src/Strategy/DateTimeFormatterStrategy.php

```diff
@@ -48,7 +48,8 @@ public function __construct($format = DateTime::RFC3339, DateTimeZone $timezone
     public function extract($value)
     {
         if ($value instanceof DateTime) {
-            return $value->format($this->format);
+            $extractionFormat = preg_replace('/(?<![\\\\])[+|!\*]/', '', $this->format);
```

Comment:
For those reviewing the issue later to understand what it does, the above matches an of the characters `+|!*` NOT preceded by a `\\`.